### PR TITLE
auto layout

### PR DESCRIPTION
**Pull request by belchercd**

### CODE REVIEW

File: fbuapp/View Controllers/EditViewController.m [hunk 1]
```diff
@@ -85,27 +91,16 @@ - (void)fetchInfo {
...
-    self.editEmail.layer.borderColor = [UIColor lightGrayColor].CGColor;
-    self.editEmail.layer.cornerRadius = 5.0f;
+- (void)formatting:(UITextField *)textField {
+    textField.layer.borderWidth = 0.5f;
+    textField.layer.borderColor = [UIColor lightGrayColor].CGColor;
+    textField.layer.cornerRadius = 5.0f;
 }
 
 
+
```
Comment (eazheng) [inline fbuapp/View Controllers/EditViewController.m]: delete extra spaces

File: fbuapp/View Controllers/EditViewController.m [hunk 2]
```diff
@@ -13,20 +13,26 @@
 
 @interface EditViewController ()
 
+
```
Comment (eazheng) [inline fbuapp/View Controllers/EditViewController.m]: delete extra spacing

File: fbuapp/View Controllers/EditViewController.m [hunk 3]
```diff
@@ -13,20 +13,26 @@
...
     self.navigationItem.rightBarButtonItem = [[UIBarButtonItem alloc] initWithTitle:@"Save" style:UIBarButtonItemStylePlain target:self action:@selector(didTapSave)];
     
     self.navigationItem.title = [NSString stringWithFormat:@"Edit Profile"];
     
-    [self formatting];
+    [self formatting:self.editFirstName];
+    [self formatting:self.editLastName];
+    [self formatting:self.editUsername];
+    [self formatting:self.editBio];
+    [self formatting:self.editEmail];
```
Comment (eazheng) [inline fbuapp/View Controllers/EditViewController.m:32]: Yay

File: fbuapp/View Controllers/ProfileViewController.m [hunk 4]
```diff
@@ -107,9 +108,9 @@ -(void)viewWillAppear:(BOOL)animated {
 
 //load user's posts to their profile
 - (void)fetchPosts {
-    if(self.mainSegment.selectedSegmentIndex == 0)
+    if(self.segmentedControl.selectedSegmentIndex == 0)
     {
-        [self checkCount];
+//        [self checkCount];
```
Comment (eazheng) [inline fbuapp/View Controllers/ProfileViewController.m]: delete this comment

File: fbuapp/View Controllers/ProfileViewController.m [hunk 5]
```diff
@@ -120,15 +121,16 @@ - (void)fetchPosts {
...
+                [self checkCount];
                 NSLog(@"%@", posts);
                 [self.feed reloadData];
             }
         }];
         [self.feed.refreshControl endRefreshing];
     }
-    else if(self.mainSegment.selectedSegmentIndex == 1) {
+    else if(self.segmentedControl.selectedSegmentIndex == 1) {
+//        [self checkCount];
```
Comment (eazheng) [inline fbuapp/View Controllers/ProfileViewController.m]: delete this comment

File: fbuapp/Views/PostTableView.m [hunk 6]
```diff
@@ -120,6 +120,8 @@ - (UITableViewCell *)tableView:(UITableView *)tableView
     cell.post = post;
     cell.currentUserId = self.currentUserId;
     cell.eventAuthor.text = [PFQuery getUserObjectWithId: post[@"eventAuthor"]][@"firstName"];
+    
+
```
Comment (eazheng) [inline fbuapp/Views/PostTableView.m]: delete extra space

File: fbuapp/View Controllers/EditViewController.h [hunk 7]
```diff
@@ -18,6 +18,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (weak, nonatomic) IBOutlet UITextView *editBio;
 @property (weak, nonatomic) IBOutlet UITextField *editEmail;
 
+
```
Comment (eazheng) [inline fbuapp/View Controllers/EditViewController.h]: remove extra space

File: fbuapp/View Controllers/ProfileViewController.m (hunk 4 above)
Comment (danthecodingman) [inline fbuapp/View Controllers/ProfileViewController.m]: remove these comments

**Review by danthecodingman — COMMENTED** [review]
Remove some comments

**Pull request merged**